### PR TITLE
chore:dashboard layout

### PR DESCRIPTION
**Pull request by arielrecto**

### CODE REVIEW

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<script lang="ts">
+	import { ButtonSize } from './../../../../lib/components/Base/Button/types.ts';
+	import {
+		FontWeight,
+		TextColor,
+		TextSize,
+		TextTag,
+	} from '$lib/components/Base/Text/types.ts'
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]:
```suggestion
	} from '$lib/components/Base/Text/types'
```

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+<JoyContainer
+	class="w-full overflow-y-auto"
+	col={true}
+	padding={ContainerPadding.SM}
+	gap={ContainerGap.Y_LG}
+>
+	<JoyContainer class="w-full" padding={ContainerPadding.SM}>
+		<JoyContainer col={true}>
+			<JoyText tag={TextTag.PARA} size={TextSize.XS}>Thursday, August 29</JoyText>
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use luxon for dynamic dates. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+			<JoyText
+				tag={TextTag.H1}
+				weight={FontWeight.BOLD}
+				size={TextSize.XL_3}
+				color={TextColor.PRIMARY}>Hi John</JoyText
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto, can we use the `user` store from `modules/authentication` for the username. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+	</JoyContainer>
+
+	<div class="grid grid-cols-2 grid-flow-row gap-10 w-full">
+		<JoyContainer
+			col={true}
+			rounded={BorderRounded.XL}
+			padding={ContainerPadding.SM}
+			border={Border.XXS}
+		>
+			<JoyText tag={TextTag.PARA} size={TextSize.SM}>Completed Tasks</JoyText>
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+			>
+			<JoyText tag={TextTag.H1} size={TextSize.XL_6} weight={FontWeight.BOLD}>0</JoyText>
+		</JoyContainer>
+		<JoyContainer
+			col={true}
+			rounded={BorderRounded.XL}
+			padding={ContainerPadding.SM}
+			border={Border.XXS}
+		>
+			<JoyText tag={TextTag.PARA} size={TextSize.SM}>Upcoming Tasks</JoyText>
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+			>
+			<JoyText tag={TextTag.H1} size={TextSize.XL_6} weight={FontWeight.BOLD}>0</JoyText>
+		</JoyContainer>
+		<JoyContainer
+			col={true}
+			rounded={BorderRounded.XL}
+			padding={ContainerPadding.SM}
+			border={Border.XXS}
+		>
+			<JoyText tag={TextTag.PARA} size={TextSize.SM}>Due Today</JoyText>
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+			<JoyText tag={TextTag.H1} size={TextSize.XL_6} weight={FontWeight.BOLD}>0</JoyText>
+		</JoyContainer>
+
+		<JoyContainer
+			col={true}
+			rounded={BorderRounded.XL}
+			padding={ContainerPadding.SM}
+			border={Border.XXS}
+		>
+			<JoyText tag={TextTag.PARA} size={TextSize.SM}>Overdue Tasks</JoyText>
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+
+	<JoyContainer
+		class="w-full"
+		col={true}
+		border={Border.XXS}
+		rounded={BorderRounded.XL}
+		padding={ContainerPadding.MD}
+	>
+		<JoyText tag={TextTag.H1} weight={FontWeight.BOLD} size={TextSize.SM}
+			>Recent Project(s)</JoyText
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+		>
+		<JoyContainer>
+			<span class="w-12 aspect-square rounded-lg bg-gray-400"> </span>
+			<JoyContainer col={true}>
+				<JoyText tag={TextTag.H1} size={TextSize.XS} weight={FontWeight.BOLD}>
+					Collab Learn
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization 'common.platform.name' for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+				</JoyText>
+				<JoyText tag={TextTag.PARA} size={TextSize.XS}>Web-based Platform for Project and Collaboration</JoyText>
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+			</JoyContainer>
+           
+		</JoyContainer>
+        <JoyButton class="w-full" variant={ButtonVariant.PRIMARY} size={ButtonSize.MD}>
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+	</JoyContainer>
+    <JoyContainer
+		class="w-full"
+		col={true}
+		border={Border.XXS}
+		rounded={BorderRounded.XL}
+		padding={ContainerPadding.MD}
+	>
+		<JoyText tag={TextTag.H1} weight={FontWeight.BOLD} size={TextSize.SM}
+			>Recent Project(s)</JoyText
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+		>
+		<JoyContainer>
+			<span class="w-12 aspect-square rounded-lg bg-gray-400"> </span>
+			<JoyContainer col={true}>
+				<JoyText tag={TextTag.H1} size={TextSize.XS} weight={FontWeight.BOLD}>
+					Collab Learn
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+				</JoyText>
+				<JoyText tag={TextTag.PARA} size={TextSize.XS}>Web-based Platform for Project and Collaboration</JoyText>
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+			</JoyContainer>
+           
+		</JoyContainer>
+        <JoyButton class="w-full" variant={ButtonVariant.PRIMARY} size={ButtonSize.MD}>
+            See Project(s)
```
Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]: Hello @arielrecto , can we use localization for this label. Thanks!

File: src/routes/(authenticated)/app/dashboard/+page.svelte [hunk 16]
```diff
@@ -0,0 +1,141 @@
+<script lang="ts">
+	import { ButtonSize } from './../../../../lib/components/Base/Button/types.ts';
```
[the remark this answers is not in the document]
Comment (arielrecto) [reply src/routes/(authenticated)/app/dashboard/+page.svelte]: hi @weave-dev the changes is applied

Comment (weave-dev) [inline src/routes/(authenticated)/app/dashboard/+page.svelte]:
```suggestion
	import { ButtonSize } from '$lib/components/Base/Button/types.ts';
```

File: src/routes/(authenticated)/app/dashboard/+page.svelte (hunk 8 above)
Comment (weave-dev) [reply src/routes/(authenticated)/app/dashboard/+page.svelte]: Hi @arielrecto, please update last remaining change. Thanks

**Review by weave-dev — DISMISSED** [review]

**Review by weave-dev — APPROVED** [review]

**Review by weave-dev — DISMISSED** [review]
LGTM

**Review by weave-dev — APPROVED** [review]

**Comment by weave-dev** [comment]
@arielrecto please create a new PR. thanks

**Pull request closed without merging**